File: app/src/main/java/com/example/intern_anrdoid_2020/view/MainActivity.java
package com.example.intern_anrdoid_2020.view;

import androidx.appcompat.app.ActionBar;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentTransaction;

import android.os.Bundle;

import com.example.intern_anrdoid_2020.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        showLoginFragment();

        ActionBar actionBar = getSupportActionBar();
        actionBar.setTitle(R.string.login_view);
    }

    private void showLoginFragment() {
        FragmentTransaction fragmentTransaction = getSupportFragmentManager().beginTransaction();
        fragmentTransaction.replace(R.id.main_layout_frame, new LoginFragment())
                           .addToBackStack(null)
                           .commit();
    }
}

File: app/src/main/java/com/example/intern_anrdoid_2020/view/MainActivity.kt
package com.example.intern_anrdoid_2020.view

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.intern_anrdoid_2020.R

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        showLoginFragment()
        val actionBar = supportActionBar
        actionBar?.setTitle(R.string.login_view)
    }

    private fun showLoginFragment() {
        val fragmentTransaction = supportFragmentManager.beginTransaction()
        fragmentTransaction.replace(R.id.main_layout_frame, LoginFragment())
                .addToBackStack(null)
                .commit()
    }
}
